libraries/featureflag/api/src/main/kotlin/io/element/android/libraries/featureflag/api/FeatureFlags.kt

```diff
@@ -175,4 +175,13 @@ enum class FeatureFlags(
         // False so it's displayed in the developer options screen
         isFinished = false,
     ),
+    SharePos(
+        key = "feature.share_pos",
+        title = "Share pos",
```

Comment:
Maybe the title should include "Sliding sync" or "Sync"?

Reply:
https://github.com/element-hq/element-x-android/pull/5010/commits/28e90d6fd547ed6caec4261ca4aa5fb5ed832e1a